pepper-apis/src/main/resources/db-changes/schema/DDP-5045-sendgrid-event-message-id-nullable.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="yufeng" id="20210309-sendgrid-event-sg-message-id-column-nullable">
+        <dropNotNullConstraint tableName="sendgrid_event" columnName="sg_message_id" columnDataType="varchar(100)"/>
```

Comment:
Noticed an issue while looking through the logs:

![Screen Shot 2021-03-09 at 6 34 06 PM](https://user-images.githubusercontent.com/5713833/110553989-ee6ff180-8107-11eb-9568-01bd209ff664.png)

Looking through the [docs](https://sendgrid.com/docs/for-developers/tracking-events/event/#json-objects), it appears that `sg_message_id` can be null, so I'm dropping the null constraint here.
> sg_message_id - a unique, internal SendGrid ID for the message. The first half of this ID is pulled from the smtp-id. The message ID will be included in most cases. In the event of an asynchronous bounce, the message ID will not be available.